packages/grafana-runtime/src/config.ts

```diff
@@ -87,6 +87,8 @@ export class GrafanaBootConfig implements GrafanaConfig {
   caching = {
     enabled: false,
   };
+  defaultBaseLayer: any;
+  disableCustomBaseLayer = false;
```

Comment:
rather than define this here can we put it at the bottom of:
https://github.com/grafana/grafana/blob/main/packages/grafana-data/src/types/config.ts#L127

with real type `MapLayerOptions`

both should be optional and have geomap prefix